src/rest/orderRest.js

```diff
@@ -0,0 +1,14 @@
+import { axiosInstance } from '../config';
+
+const api = {
+  fetchOrdersByPatient: patientUUID => axiosInstance.get(`order?totalCount=true&sort=desc&status=active&patient=${patientUUID}&v=full`)
```

Comment:
I just noticed that this restricts to status=active, so let's change the name to fetchActiveOrdersByPatient